datadog/threadstats/base.py

```diff
@@ -289,7 +302,7 @@ def flush(self, timestamp=None):
             self._is_flush_in_progress = True
 
             # Process metrics
-            metrics = self._get_aggregate_metrics(timestamp or time())
+            (metrics, dists) = self._get_aggregate_metrics_and_dists(timestamp or time())
```

Comment:
this is a bit ugly but not sure if there's a better approach

Reply:
Looks OK to me. Parens around `(metrics, dists)` are not needed.

```suggestion
            metrics, dists = self._get_aggregate_metrics_and_dists(timestamp or time())
```